tests/integration/native-image/se-1/src/main/java/io/helidon/tests/integration/nativeimage/se1/Se1Main.java

```diff
@@ -40,6 +43,8 @@
  * Main class of this integration test.
  */
 public final class Se1Main {
+    static LazyValue<WebServer> server;
```

Comment:
Why?

Reply:
Rather obscure way how to get current webserver port inside the service, removed and simplified